lib/rubocop/cop/util.rb

```diff
@@ -148,6 +148,10 @@ def move_pos(src, pos, step, condition, regexp)
         pos
       end
 
+      def looking_at(range, regexp)
+        range.source_buffer.source.match(/\G#{regexp}/, range.begin_pos)
```

Comment:
@jonas054 Whenever an offset is internally given in regexp matching, `\G` takes effect and matches that point.  I think I'll improve the official documentation later.